src/components/Dashboard/ActionRequired.vue

```diff
@@ -45,3 +45,12 @@ export default class ActionRequired extends Vue {
   padding: 0 0 16px !important;
 }
 </style>
+
+<style lang="scss" scoped>
+
+.details-btn {
+  margin-bottom: 0.25rem;
+  height: inherit !important;
+  border: none;
+}
```

Comment:
Why the special case for this button?

Reply:
If you need to "pull this button up" so it doesn't affect the linespacing, the pull the button up with negative top margin.

Or maybe make this button have less padding overall (but it will be different from the other buttons).

Special case are not good!

Reply:
In todolist also we have the button with an outline (instead of text) and border none. 
As mentioned i removed height and did with negative top margin

Reply:
OK, thanks.